index.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module 'express-promise-router' {
```

Comment:
You are importing `Router` to the scope, and then redefining a `Router` function, which at least on my version of typescript leads to a conflict

Reply:
I'm using 2.5.3,and works fine.

Reply:
I tried to upgrade to 2.6.2, and the problem still maintains itself:
`node_modules/express-promise-router/index.d.ts(2,13): error TS2440: Import declaration conflicts with local declaration of 'Router'.`.

Still, the fix is pretty simple:
```
import {Router as RouterExpress, RouterOptions} from 'express'

     function Router(options?: RouterOptions): RouterExpress
 
     export = Router
 }
```

What do you think?